app/actions/remote/channel.ts

```diff
@@ -49,6 +51,98 @@ export type MyChannelsRequest = {
     error?: unknown;
 }
 
+export type ChannelMembersRequest = {
+    members?: ChannelMembership[];
+    error?: unknown;
+}
+
```

Comment:
To each of the `catch` clause, add the `logError` function so as to track down errors via Sentry.

Reply:
Done.  Should we consider adding a parameter to `forceLogoutIfNecessary` that includes the function name and add the logError in that function?

Reply:
Added the suggestion in mattermost and cc'd you.